core/src/main/scala/spire/math/Interval.scala

```diff
@@ -421,6 +423,9 @@ sealed abstract class Interval[A](implicit order: Order[A]) { lhs =>
       case (_, _) => lhs * rhs.reciprocal
     }
 
+  def /(rhs: A)(implicit ev: Field[A]): Interval[A] =
+    lhs * rhs.reciprocal
```

Comment:
I could have written `lhs / Interval.point(rhs)` but if you read that method's body I've just inlined it here (without the `point` call).

We don't currently have a *direction division* implementation for `Interval[_]` (the last one I wrote was a bit buggy and using reciprocal was easier to prove correct).